lib/process-services-cloud/src/lib/task/task-form/components/task-form-cloud.component.spec.ts

```diff
@@ -120,88 +122,106 @@ describe('TaskFormCloudComponent', () => {
 
     describe('Claim/Unclaim buttons', () => {
 
-        it('should not show release button for standalone task', async () => {
+        it('should not show release button for standalone task', () => {
             component.taskId = 'task1';
             component.loadTask();
             getTaskSpy.and.returnValue(of(taskDetails));
 
             fixture.detectChanges();
-            await fixture.whenStable();
```

Comment:
why are y removing it ?

Reply:
we don't need that in that tests which doesn't use any async operations, animations and browser macroTasks (setTimeout, requestAnimationFrame, etc.)